<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWdgUi</class>
 <widget class="QWidget" name="ControlWdgUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>78</height>
       </rect>
      </property>
      <property name="title">
       <string>Estado</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>571</width>
        <height>64</height>
       </rect>
      </property>
      <property name="title">
       <string>Comparador de velocidades</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>N=0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>BRK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>EXC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>SYNC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>LOVS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>HOVS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Conexiones</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Frecuencia</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="intValue" stdset="0">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="value" stdset="0">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="value" stdset="0">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLCDNumber" name="lcdNumber_4">
         <property name="intValue" stdset="0">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Potencia Activa</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_5">
         <property name="value" stdset="0">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLCDNumber" name="lcdNumber_6">
         <property name="intValue" stdset="0">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Arranque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Parada</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Descarga</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Tipo regulacion</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="cmdRegTypeP">
              <property name="text">
               <string>Potencia</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdRegTypeA">
              <property name="text">
               <string>Apertura</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Disparo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frame_6">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="lbIndRem">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>REM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbIndLoc">
           <property name="text">
            <string>LOC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
